src/vs/workbench/contrib/issue/electron-sandbox/issue.contribution.ts

```diff
@@ -57,16 +54,6 @@ class NativeIssueContribution extends BaseIssueContribution {
 			});
 		};
 
-		Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).registerConfiguration({
-			properties: {
-				'issueReporter.experimental.auxWindow': {
```

Comment:
since we are just removing this setting (and it was experimental), I don't think we need to do any migration or such?